### ISSUE TITLE
MathNet.Numerics.Signed v4.8 not passing sn verification

### ISSUE THREAD

**Issue by fbordignon** [mathnet/mathnet-numerics#643]
Version 4.8 of MathNet.Numerics.Signed

```
sn -v MathNet.Numerics.dll

Microsoft (R) .NET Framework Strong Name Utility  Version 4.0.30319.0
Copyright (c) Microsoft Corporation.  All rights reserved.

MathNet.Numerics.dll does not represent a strongly named assembly
```
Running sn -v on version 4.7 yields "Assembly 'MathNet.Numerics.dll' is valid"

This affects my application, that does not allow for loading assemblies that are not strongly named

**Comment by fbordignon** [comment]
ps. this is using NuGet binaries.

**Comment by cdrnet** [comment]
Yes, for some reason it seems only the net40 is strong named. I'll need to do a new release. Thanks for reporting.

**Comment by cdrnet** [comment]
Should be fixed in v4.8.1. Please try again with that version.

**Comment by fbordignon** [comment]
Confirmed that it is fixed. kudos :clap: :clap: :clap:
